File: OMWAY/src/main/java/com/example/omway/dto/payment/CashDto.java
package com.example.omway.dto.payment;

import com.example.omway.model.payment.PaymentMethod;
import lombok.Data;

@Data
public class CashDto {
    private Integer paymentId;
    private double total;
    private PaymentMethod paymentMethod;
    private Integer rideId;
}

File: OMWAYAppViews/OMWayApp/app/src/main/java/com/main/omwayapp/apirest/dto/payment/cash/CashDto.kt
package com.main.omwayapp.apirest.dto.payment.cash

import android.os.Parcelable
import kotlinx.parcelize.Parcelize
import kotlinx.serialization.Serializable

@Parcelize
@Serializable
data class CashDto(
    val paymentId: Int,
    val total: Double,
    val paymentMethod: String,
    val rideId:Int
) : Parcelable
